File: forge/src/main/java/dev/aika/bigclever/forge/BigCleverPlatformImpl.java
package dev.aika.bigclever.forge;

import net.minecraftforge.fml.loading.FMLPaths;

import java.nio.file.Path;

public class BigCleverPlatformImpl {
    public static Path getGameDir() {
        return FMLPaths.GAMEDIR.get();
    }
}

File: neoforge/src/main/kotlin/dev/aika/bigclever/neoforge/BigCleverPlatformImpl.kt
@file:JvmName("BigCleverPlatformImpl")

package dev.aika.bigclever.neoforge

import net.neoforged.fml.loading.FMLPaths
import java.nio.file.Path

@Suppress("unused")
fun getGameDir(): Path = FMLPaths.GAMEDIR.get()
